flask_resty/testing.py

```diff
@@ -30,30 +30,37 @@ def get_meta(response):
 def assert_value(actual, expected):
     if isinstance(expected, Mapping):
         assert isinstance(actual, Mapping)
-        for k, v in expected.items():
-            assert_value(actual.get(k, None), v)
+        # Unlike all the others, this checks that the actual items are a
+        # superset of the expected items, rather than that they match.
+        for key, value in expected.items():
+            if value is not UNDEFINED:
```

Comment:
you could have had (lookng at the old code) `assert_value(actual.get(k, UNDEFINED), v)`, so that if v is UNDEFINED, the old code would just behave like this (might be missing something)